x-pack/plugins/alerting/server/application/maintenance_window/transforms/transform_maintenance_window_attributes_to_maintenance_window.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This should not like to a version?

Reply:
This one doesn't need to be since it's transforming between application and data layer, not the route layer

Reply:
I heard that we are going to have to do it to protect ourself over when we have kiban N and Kibana N+1 running against each other on the server-less. Another fight for another day.